bigchaindb_driver/crypto.py

```diff
@@ -21,4 +24,4 @@ def generate_keypair():
 
     """
     return CryptoKeypair(
-        *(k.decode() for k in crypto.ed25519_generate_key_pair()))
+        *(k.decode() for k in crypto.ed25519_generate_key_pair(seed)))
```

Comment:
This is failing in the unit tests (run by Travis CI). They key line in the error output is:
```text
TypeError: ed25519_generate_key_pair() takes 0 positional arguments but 1 was given
```
I think this can be fixed by changing it to say `...pair(seed=seed)))`

Reply:
Won't it be fixed automatically once cryptocondition package is updated

Reply:
Yes, you were right, sorry. I got ahead of myself. Doh!